csrc/quantization/machete/generate.py

```diff
@@ -0,0 +1,444 @@
+import itertools
+import math
+import os
+import shutil
+from collections.abc import Iterable
+from dataclasses import dataclass
+from typing import List, Optional, Tuple, Union
+
+import jinja2
```

Comment:
does this need to be added to requirements-build.txt?

Reply:
good call, done